bounce/server/resource/__init__.py

```diff
@@ -70,16 +70,42 @@ def validate(request_cls, response_cls):
             body should match
     """
 
+    def set_defaults(schema, info, parent_key=None):
+        # if there is no value specified for the keys
+        # set the default values if there are
+        # corresponding default values found in the json schema
+        import pdb
+        pdb.set_trace()
+        for key in schema:
+            value = schema[key]
+            if key == 'default':
+                if parent_key not in info:
+                    info.update({parent_key: value})
+                elif not info[parent_key]:
+                    info.update({parent_key: value})
+            if isinstance(value, dict):
+                sub_schema = value
+                parent_key = key
+                set_defaults(sub_schema, info, parent_key)
+        return info
+
     # pylint: disable=missing-docstring
     def decorator(coro):
         @wraps(coro)
         async def wrapper(endpoint, request, *args, **kwargs):
             if hasattr(request_cls, '__body__'):
                 # Return a 400 if the request body does not meet the
                 # required schema
+                # Body values come as arrays of length 1 so turn
+                # them into single values
+                body_no_defaults = {
+                    key: request.form.get(key)
+                    for key in request.form
+                }
```

Comment:
Do you need to initialize this object like this, or does this work?

```python
body_no_defaults = request.json
```

Reply:
Oh I think in postman I was sending and receiving form-data instead of json my bad.  Request.json works when you specify the data you want to send in postman as json and not form-data:
![image](https://user-images.githubusercontent.com/7409729/47685894-94ed7480-db95-11e8-8520-b17771a798b4.png)